src/github/pullRequestManager.ts

```diff
@@ -1534,6 +1542,14 @@ export class PullRequestManager {
 
 		return events;
 	}
+
+	createGitHubRepository(remote: Remote, credentialStore: CredentialStore): GitHubRepository {
+		return new GitHubRepository(remote, credentialStore);
+	}

```

Comment:
I believe sinon has the ability to mock a constructor, but it always feels too invasive for my tastes. (What if there's another instantiation somewhere on the stack that I _don't_ want to mock?)